Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "bagua-net"
+version = "0.1.0"
+authors = ["shjwudp"]
+edition = "2018"
```

Comment:
add MIT license, and put your full name in authors (with email address)

Reply:
DONE